sacred/randomness.py

```diff
@@ -12,15 +12,22 @@
 def get_seed(rnd=None):
     if rnd is None:
         return random.randint(*SEEDRANGE)
-    return rnd.randint(*SEEDRANGE)
+    else:
+        try:
+            return rnd.integers(*SEEDRANGE, dtype=int)
+        except Exception:
+            return rnd.randint(*SEEDRANGE)
 
 
-def create_rnd(seed):
+def create_rnd(seed, np_legacy=False):
```

Comment:
You can directly access the setting from `SETTINGS.CONFIG.NUMPY_RANDOM_LEGACY_API` here. Is there a reason for the argument? If I saw this correctly, it is always set to `SETTINGS.CONFIG.NUMPY_RANDOM_LEGACY_API`. Or do you want to allow setting the API version independently for every captured function?